src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Figure figure = new Circle();
+        int figureNumber = (int) (1 + Math.random() * (6 - 1));
+        switch (figureNumber) {
+            case 1:
+                figure = new Circle(Math.random() * 10);
+                break;
+            case 2:
+                figure = new IsoscelesTrapezoid(Math.random() * 10,
+                        Math.random() * 10,
+                        Math.random() * 10);

```

Comment:
it's a little bit unreadable. let's create private methods for each figure creation and explicitly create variables to pass to a constructor